core/src/main/java/org/apache/carbondata/core/datastore/chunk/AbstractRawColumnChunk.java

```diff
@@ -47,6 +48,14 @@
 
   private DataChunk3 dataChunkV3;
 
+  private PresenceMeta[] presenceMeta;
+
+  private boolean isLVSeperated;
+
+  private boolean isAdaptiveForDictionary;
```

Comment:
is it required for both dictionary and nodictionary? since its an abstract class we can just add a property to know whether its adaptive or not in base class ? the child is anyways DimensionRawColumnChunk